### PR TITLE
Add mmapv1.json for replica sets and sharded configurations

### PR DESCRIPTION
**Pull request by jyemin**
I haven't had time to actually test these... @prashantmital can you take them for a spin since you're in progress on the ticket?

### CODE REVIEW

File: .evergreen/orchestration/configs/replica_sets/mmapv1.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          "dc": "pa"
+        }
+      }
+    },
+    {
+      "procParams": {
+        "ipv6": true,
+        "bind_ip": "127.0.0.1,::1",
+        "journal": true, 
+        "storageEngine": "mmapv1",
```
Comment (ShaneHarvey) [inline .evergreen/orchestration/configs/replica_sets/mmapv1.json]: Is storageEngine valid for an arbiter?

Comment (jyemin) [reply .evergreen/orchestration/configs/replica_sets/mmapv1.json]: Probably not.  I'll remove

Comment (jyemin) [reply .evergreen/orchestration/configs/replica_sets/mmapv1.json]: Done

**Comment by prashantmital** [comment]
I can confirm that the config works.

**Comment by prashantmital** [comment]
UPDATE: I also confirmed that the sharded config works (see https://evergreen.mongodb.com/version/5d4b6d9d0ae6061b78a24d37).

**Pull request closed without merging**